src/components/modals/modification/RotateScale.vue

```diff
@@ -18,9 +18,9 @@ function modify() {
       <h2>Rotate</h2>
     </template>
     <template #body>
-      <p>Rotates the mode of your scale.</p>
-      <p>The resulting scale will be sort ascendingly.</p>
       <div class="control-group">
+        <p>Rotates the mode of your scale.</p>
+        <p>The resulting scale will be sorted ascendingly.</p>
```

Comment:
I copied this over from SW1 and I thinks it's the first time I've seen the word *ascendingly*. I'd rather replace it with *in ascending order*.